templates/vhost/_setenv.epp

```diff
@@ -0,0 +1,17 @@
+<% unless $setenv.empty { -%>
```

Comment:
Can we move the `empty()` check to `vhost.pp` and only include the concat fragment if it's non-empty? In fact, we already have `$use_env_mod = !empty($setenv)` so you can reuse that variable.

Reply:
Yeah, I saw that but kept it as it was before.  I will add a commit to avoid the double-test.